src/pages/landing/weekend-trading/components/_hero.js

```diff
@@ -0,0 +1,221 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import { Flex, Container, Show } from 'components/containers'
+import { Header } from 'components/elements'
+import { localize } from 'components/localization'
+import { Background } from 'components/elements/background-image'
+import { LinkButton } from 'components/form'
+import device from 'themes/device.js'
+
+const BackgroundWrapper = styled(Background)`
+    background-size: cover;
+    background-position: bottom right;
+    @media ${device.desktopL} {
+        height: 70rem !important;
+    }
+    @media ${device.desktop} {
+        height: 65rem !important;
+    }
+    @media ${device.tabletL} {
+        height: 155rem !important;
+    }
+    @media ${device.tablet} {
+        height: 136rem !important;
+    }
+    @media ${device.tabletS} {
+        height: 105rem !important;
+    }
+    @media ${device.mobileL} {
+        height: 95rem !important;
+    }
+    @media ${device.mobileM} {
+        height: 90rem !important;
+    }
+    @media ${device.mobileS} {
+        height: 80rem !important;
+    }
+`
+
+const Wrapper = styled(Container)`
+    background-color: transparent;
+    height: unset;
+
+    @media ${device.tabletL} {
+        padding-left: 4rem;
+    }
+
+    @media ${device.mobileL} {
+        padding-left: 2rem;
+        flex-direction: column-reverse;
+        justify-content: center;
+    }
+`
+
+const InformationWrapper = styled(Flex)`
+    width: 100%;
+    max-width: 60.5rem;
+    z-index: 1;
+
+    @media ${device.tabletL} {
+        max-width: 55rem;
+        top: 280px;
+        padding: 0 16px;
+    }
+    @media ${device.mobileL} {
+        max-width: 328px;
+    }
+    @media ${device.mobileM} {
+        padding: 0;
+    }
+`
+
+const HeroContent = styled(Flex)`
+    height: unset;
+
+    ${Header} {
+        font-size: 24px;
+        font-weight: 200;
+        color: var(--color-black-3);
+        display: flex;
+        margin-top: 1rem;
+        line-height: 2.6rem;
+        max-width: 90%;
+    }
+    @media ${device.laptopM} {
+        ${Header} {
+            font-size: 22px;
+        }
+    }
+    @media ${device.tabletL} {
+        ${Header} {
+            font-size: 20px;
+            line-height: 26px;
+            max-width: 100%;
+        }
+    }
+
+    @media ${device.mobileL} {
+        ${Header} {
+            font-size: 18px;
+            margin-top: 10px;
+            margin-bottom: 0;
+        }
+    }
+    @media ${device.mobileS} {
+        ${Header} {
+            max-width: 98%;
+        }
+    }
+`
```

Comment:
Please use the Header `type` for these. If the designs are in Figma, they should be following their specifications based on typography.js.

Reply:
Same for all other Header.

Reply:
done